.travis.yml

```diff
@@ -28,10 +30,8 @@ env:
   - CXX=g++-4.8 DB="maria mysql mysql2 postgres sqlite3 oracledb" KNEX_TEST_TIMEOUT=60000 ORACLE_HOME=/u01/app/oracle/product/11.2.0/xe ORACLE_SID=XE OCI_LIB_DIR=/u01/app/oracle/product/11.2.0/xe/lib LD_LIBRARY_PATH=/u01/app/oracle/product/11.2.0/xe/lib
 
 addons:
-  postgresql: '9.4'
+  postgresql: '9.6'
```

Comment:
anyways for the release I'm going through all the recent commit history and check if there was breaking changes to decide if next version is 0.13 or 0.12.7